.evergreen/scripts/run-tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+. .evergreen/scripts/env.sh
+if [ -n "${test_encryption}" ]; then
+  ./.evergreen/hatch.sh encryption:setup &
```

Comment:
We might need to break this out the way we handle it in the go driver, as a separate background command: https://github.com/mongodb/mongo-go-driver/blob/cf0348c9a63f6e4086e2c62aa62e70a9d3b3a986/.evergreen/config.yml#L508